test/plugin/test_in_wmi.rb

```diff
@@ -7,6 +7,8 @@ class WmiInputTest < Test::Unit::TestCase
   end
 
   CONFIG = %[
+    tag monitor.wmi
+    instance_of Win32_Processor

```

Comment:
The parameter name `instance_of` isn't friendly for end users because such a term isn't appeared in Microsoft's document (maybe). Probably it's wmi-light's local naming.
`CimClassName` is better than it because the commandlet of PowerShell "Get-CimClass" uses this term.

* https://docs.microsoft.com/en-us/powershell/module/cimcmdlets/get-cimclass?view=powershell-7
* https://docs.microsoft.com/en-us/windows/win32/wmisdk/common-information-model

```powershell
> Get-CimClass Win32_Processor


   NameSpace: ROOT/cimv2

CimClassName                        CimClassMethods      CimClassProperties
------------                        ---------------      ------------------
Win32_Processor                     {SetPowerState, R... {Caption, Description, InstallDate, Name...}
```

Reply:
Right, `instance_of` is bad name in this case.

In the context of WMI being Microsoft's implementation of CMI, `class_name` is the appropriate name to use here.

Reply:
> class_name is the appropriate name to use here.

I think so too.